framework/libra-framework/sources/ol_sources/tests/account.test.move

```diff
@@ -8,7 +8,8 @@ module ol_framework::test_account {
   use ol_framework::ol_account;
   use ol_framework::ancestry;
   use diem_framework::coin;
-  use diem_std::debug::print;
+  // use diem_std::debug::print;
```

Comment:
Could this commented line be removed?

Reply:
ugly, but I keep these in there for convenience when I need to print stuff in debug.